src/PolykeyClient.ts

```diff
@@ -88,6 +119,7 @@ class PolykeyClient {
   public readonly session: Session;
   public readonly rpcClientClient: RPCClient<typeof clientClientManifest>;
```

Comment:
Probably should be called just `rpcClient` now.

Reply:
Done.